manifests/metrics_role.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
All other manifests filenames take the form `cluster-authentication-operator_{NUM}_{RESOURCE_DESC}.yaml`, we should keep that form in order to be able to identify our manifest files in the CVO payload I believe.

Reply:
I figured I'd need to adjust these some and like figure out what number they should be at. The names are temporary while I get the rest sorted out. 
But also, we need to rename all the manifests in here to get rid of the `cluster-authentication-operator_` prefix. They show up as `0000_70_cluster-authentication-operator_cluster-authentication-operator_0...` in the release 😳